lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,53 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementList } from './endpoints';
+
+const headers = {
```

Comment:
Code Style Note: When defining a constant user all caps for the name NAME_OF_CONST.  It will clearly identify what in the code is constant and what is variable.